spec/Centipede/Filter/UrlFilterSpec.php

```diff
@@ -29,6 +24,6 @@ function it_filters_url_hash()
 
     function it_filters_trailing_slash()
     {
-        $this->filter('https://github.com/')->shouldReturn('https://github.com');
+        $this->filter('https://github.com/')->shouldReturn('https://github.com/');
```

Comment:
Because sometimes the trailing slash is important, depending on urls. Here for root it does not matter, but for deeper paths, it can matter a lot, depending on how the app is designed.